index.js

```diff
@@ -9,6 +10,32 @@ const nextButton = document.querySelector('[data-js="button-next"]');
 const pagination = document.querySelector('[data-js="pagination"]');
 
 // States
-const maxPage = 1;
-const page = 1;
+let maxPage = 1;
+let page = 1;
 const searchQuery = "";
+
+async function fetchCharacters() {
+  try {
+    const response = await fetch(
+      `https://rickandmortyapi.com/api/character?page=${page}`
+    );
+    const { info, results } = await response.json();
+    maxPage = info.pages;
+    pagination.textContent = `${page} / ${info.pages}`;
```

Comment:
As you have saved `info.pages` in `maxPage`, you should use the variable name here:
```suggestion
    pagination.textContent = `${page} / ${maxPage}`;
```